### ISSUE TITLE
Candidate Issues for stabilization sprint

### ISSUE THREAD

**Issue by scarlett2018** [microsoft/nni#5106]
We plan to have a minor sprint for stabilization, the purpose of the sprint is to work on tech debt, bugs, CICD, etc. No new features are expected. By introducing this minor sprint, we give everyone a pause for the next big release, enjoy the minor sprint =)!

https://github.com/microsoft/nni/labels/v2.9.1

Release manager: @Louis-J 
Release start date: TBD
Feature freeze date (at most 30 days): 10.14
Code freeze date: 10.17
First test package (1 week since feature freeze): 10.18
Release date (3 weeks since feature freeze): TBD 11.14

**Training Service**
- [x] p0 - Does not support non-English path ~12/10 #5164  @Lijiaoa , bugbash: complete
- [ ] p2 - ~~#5064~~

**HPO**
- [x] p0 - WebSocket connection may broken during long run ~12/10 #5166  @Lijiaoa , bugbash: complete
- [x] p1 - Make anneal optional  ~13/10 #5165 @Bonytu , bugbash: complete
- [ ] p0 - lag after long run
- [ ] p2 - ~~Assessor.trial_end() is not invoked if no metrics sent~~
- [ ] p2 - ~~Make REST timeout configurable~~

**NAS**
- [x] p0 - Dedup for evolution search #5092 @ultmaster reviewing @liuzhe-lz , bugbash: complete
- [x] p0 - #5117 #5135 @ultmaster reviewing @J-shang , bugbash: complete
- [x] p0 - [nas] fix issue introduced by the trial recovery feature #5109 ~30/9 @QuanluZhang  reviewing @ultmaster, bugbash: complete
- [x] #5119 @ultmaster  ~10/10 reviewing @QuanluZhang , bugbash: complete

**Model Compression**
- [x] p1 - [verify issue #3514, fix if there are any problems](https://github.com/microsoft/nni/issues/3514) @J-shang @Louis-J , bugbash: complete

**ModelSpeedup**
- [x] p0 - speedup rand_like_with_shape is easy to overflow when dtype=torch.int8. @J-shang  ~30/9 waiting for #5138 pr: #5140 Pruning detectron2's Faster R-CNN #4875 #5136 @J-shang ~28/9 pr: #5140 reviewing @QuanluZhang , bugbash: complete
- [ ] p2 - ~~Autoinfer can not deal with some op, sparsity cannot propagate through these ops, like torch.nn.functional.interpolate @J-shang~~ move to 3.0
- [x] p0 - torch._C.parse_schema in pytorch < 1.9.0 do not produce enough information. Writing a schema parser needed #5131  @Louis-J  ~25/9 reviewing @J-shang , bugbash: complete
- [x] p0 - a bug in speedup propagate: executing random_init the view tensors causes error. #5018 @Louis-J  reviewing
    https://pytorch.org/docs/stable/tensor_view.html#tensor-views @J-shang , bugbash: complete
- [ ] p1 - ~~a bug in speedup replace_module: #4991. @Louis-J ~10/10~~
- [ ] p2 - ~~refactor the inference process in speedup: in 'speedup support list/dict inputspeedup doesn't support list/dict input', #5107, solving #4998 #5097 #5130 @Louis-J~~
- [ ] p2 - ~~speedup doesn't support registered buffer. @Louis-J
- [ ] p2 - ~~speedup get correct masks in cpu but wrong masks in gpu. #4945 @Louis-J~~

**WebUI**
- [x] p0 - HPO experiment hyper-parameters page error. · Issue #5127 ~28/9 reviewing @liuzhe-lz , bugbash: complete
- [x] p0 - #3549 ~30/9 @Louis-J , bugbash: complete

**Comment by ultmaster** [comment]
# NAS

* #5119 
* #5117 
* #5092

**Comment by nnzs9248** [comment]
p1 - ~~a bug in speedup replace_module: https://github.com/microsoft/nni/issues/4991. @Louis-J ~10/10~~

 any progress in this？
